src/__tests__/integration.test.js

```diff
@@ -45,14 +45,14 @@ describe('Filter tests', () => {
 describe('Pricing tests', () => {
   it('will render zero when unchecked', () => {
     const app = mount(<App />)
-    expect(app.find('#total-price').text()).toBe('0')
+    expect(app.find('#total-price').text()).toBe('Total: $0')
```

Comment:
Thanks for the comments.  I pretty much went off your code when I was lost at trying to figure out how React works.  I have been doing a lot of tutorials and finding apps in git hub I can use to make apps so I can get a better understanding.

I tried the button, and it would only subtract the prices, and not add, so I went with a checkbox.  I will try again with the buttons tomorrow so it look more like a more modern app.

I don't have complete test coverage.  I only have 3 of the files tested.  I will do more this week.  I had issues with testing makeKey.  I will keep trying to work with that one.

Thanks again, and thanks for teaching this class.  I really like React.